### PR TITLE
Boot config filters

### PR DESCRIPTION
**Pull request by tombettany**
https://trello.com/c/z239QJMg/154-5-configure-the-config-txt-to-allow-the-screen-kit-to-be-the-right-way-up-by-default

* Adds support for filters in the boot config file
* Adds display rotation for the screen kit by default
* Some tests for the added functionality

cc @Ealdwulf @skarbat

### CODE REVIEW

File: kano_settings/boot_config_line.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import re
```
Comment (skarbat) [inline kano_settings/boot_config_line.py]: Perhaps a short explanation on what this module knows to do?

**Comment by skarbat** [comment]
Looks ok to me @tombettany

**Comment by tombettany** [comment]
@Ealdwulf Right, now all the tests pass :-)

Now to check no EDID, the noobs setting and ensuring that the filtered values always appear at the bottom of the config so they are not overridden.

**Comment by tombettany** [comment]
@Ealdwulf I made sure that there is a test case for the missing EDID and the noobs setting passes from the original tests. There isn't a method to ensure that settings don't get overwritten by later, unfiltered ones but this presents quite a challenge. I think that we should get this merged in so we can begin actual testing of it.

**Comment by tombettany** [comment]
@Ealdwulf Bump. I want to try to get this in as early as possible so that we have time to test it.

**Comment by Ealdwulf** [comment]
Looks okay to me :shipit:

**Pull request merged**